### PR TITLE
Minor refactor

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**
This PR intends to further simplify the installation process.

Fixes for https://github.com/kubermatic/kubermatic/issues/9484

Signed-off-by: Waleed Malik <ahmedwaleedmalik@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.envrc
+*.tar.gz
+tf.json
+*terraform.tfstate*
+.terraform
+.terraform.lock.hcl
+terraform.tfvars
+*kubeconfig
+!*kubeconfig/
```
Comment (archups) [inline .gitignore]: Add newline at the end

**Review by archups — COMMENTED** [review]
@ahmedwaleedmalik Please take of the minor comment added, rest looks fine to me. Thanks!

File: .gitignore (hunk 1 above)
Comment (ahmedwaleedmalik) [reply .gitignore]: .gitignore shouldn't have a trailing newline but I've updated it anyways.

**Review by archups — APPROVED** [review]
LGTM

**Pull request merged**